### ISSUE TITLE
Placing pocket in End dimension is weird

### ISSUE THREAD

**Issue by TheRealZeher** [TheRealZeher/DimensionalPockets-II#3]
Weird stuff happens when using pocket in end

**Comment by NPException** [comment]
The same issue was brought to my attention for DP in MC 1.7.10. I think I may have a solution to the problem, and I'll try to fix it for DP within the next week or so. I can provide you with the code I used when I'm done if you're interested.

**Comment by TheRealZeher** [comment]
Yeah sifting through code it looked like a longer standing issue. If you could that would be perfect because I can update it and allow the END again.

**Comment by NPException** [comment]
I fixed the issue in Dimensional Pockets now. The cause is that Minecraft's teleport code between dimensions forces adjustments to the player coordinates  in some cases:

- If the source dimension is The End, the player's coordinates will be set to the spawn point of the target dimension. (Because in vanilla MC the only teleport from the End to some other dimension is to the overworld)
- The player's coordinates will be multiplied with a movement factor if one is associated with the target dimension (like the Nether: 8.0)

I copied the teleport code from the `ServerConfigurationManager` and removed all modifications to the player's position: https://github.com/NPException/Dimensional-Pockets/commit/a827e461a0ed6cb81d57d50aa17fc960f54dd854

I think you should be able to solve the issue in a similar manner. 😃
